src/routes/BreedGallery.tsx

```diff
@@ -0,0 +1,18 @@
+import { useParams } from 'react-router-dom';
+import Header from '../components/Header/Header';
+import BreedPhotos from '../components/BreedPhotos/BreedPhotos';
+
+interface Params {
+    breed: string;
+}
+
+export default function BreedGallery(props: any) {
```

Comment:
No need to use props here, right?

Reply:
Totally right. This is a remnant of a previous piece of code I wrote and forgot to remove.